### ISSUE TITLE
NTR: X-ray: subchondral fracture (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9015]
New term request X-ray: subchondral fracture (seidel_190122165502):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000287
- label: Subchondral fracture
- parents: [Fracture type (HP:4000042)](https://hpo.jax.org/app/browse/term/HP:4000042)
- pmid: 
- def: A type of fracture that occurs in the trabecular cancellous bone just beneath the subchondral bone plate without disruption of the articular surface.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9015
- synonyms:
